_data/master/navbars/reference.yml

```diff
@@ -112,6 +112,26 @@ toc:
     path: /reference/public-cloud/gce
   - title: IBM Cloud
     path: /reference/public-cloud/ibm
+- title: Host endpoints
+  section:
+    - title: Overview
+      path: /reference/host-endpoints/
+    - title: Creating policy for basic connectivity
+      path: /reference/host-endpoints/connectivity
+    - title: Creating host endpoint objects
+      path: /reference/host-endpoints/objects
+    - title: Selector-based policies
+      path: /reference/host-endpoints/selector
+    - title: Failsafe rules
+      path: /reference/host-endpoints/failsafe
+    - title: Pre-DNAT policy
+      path: /reference/host-endpoints/pre-dnat
+    - title: Apply on forwarded traffic
+      path: /reference/host-endpoints/forwarded
+    - title: Summary
```

Comment:
I would have expected the summary page to be the last one.

Reply:
The last topic is a troubleshooting item. This moving exercise is to get the "old" content out from Security into reference as holding place. Casey needs to go through this content and pick out what is still good. About 80% of this will go away at some point.